src/Main/LedBar.cpp

```diff
@@ -110,6 +110,17 @@ void LedBar::setColor(uint8_t red, uint8_t green, uint8_t blue) {
   }
 }
 
+/**
+ * @brief Sets the hue of all LEDs
+ *
+ * @param hue Hue of the color as 16-bit number
+ */
+void LedBar::setHue(uint16_t hue) {
```

Comment:
Should we also support to set saturation and brightness, or is this not of interest?